src/aleph/vm/pool.py

```diff
@@ -250,6 +250,9 @@ async def _load_persistent_executions(self):
 
                 self._schedule_forget_on_stop(execution)
 
+                # Start the snapshot manager for the VM
+                await self.snapshot_manager.start_for(vm=execution.vm)
```

Comment:
```suggestion
                # Start the snapshot manager for the VM
                if execution.vm and execution.vm.support_snapshot and self.snapshot_manager:
                    await self.snapshot_manager.start_for(vm=execution.vm)
```
The same control should be added here.